docs/reference/manual/pr_create.md

```diff
@@ -15,6 +15,7 @@ doing pr create 1234 --draft -r "john.doe@company.com jane.doe@company.com"
 doing pr create 1234 --draft -r "@me jane.doe@company.com"
 doing pr create 1234 --draft --checkout
 doing pr create 1234 --delete-source-branch --self-approve --auto-complete
+doing pr create 1234 --default-branch develop
 ```
 
 !!! notes ""
```

Comment:
Line 25 should also be updated to let users know of this new option.

Reply:
updated